habitat-lab/habitat/datasets/rearrange/configs/hssd_ycb_multi_goal.yaml

```diff
@@ -0,0 +1,158 @@
+---
+    dataset_path: "data/fpss/hssd-hab-uncluttered.scene_dataset_config.json"
+    additional_object_paths:
+    - data/objects/ycb/configs/
+    scene_sets:
+      - name: "train"
+        included_substrings:
+          - "102344049"
+          - "102344193"
+          - "102344250"
+          - "102344280"
+          - "102816216"
+          - "102817200"
+          - "103997424_171030444"
+          - "103997460_171030507"
+          - "103997919_171031233"
+          - "104348082_171512994"
+          - "104348463_171513588"
+          - "104862621_172226772"
+          - "104862639_172226823"
+          - "104862660_172226844"
+          - "104862681_172226874"
+          - "105515211_173104179"
+          - "105515448_173104512"
+          - "106366173_174226431"
+          - "106878945_174887058"
+          - "107734449_176000403"
+          - "107734479_176000442"
+          - "108294573_176710113"
+          - "108294870_176710551"
+          - "108294897_176710602"
+          - "108736635_177263256"
+          - "108736737_177263406"
+          - "108736824_177263559"
+          - "108736851_177263586"
+          - "108736872_177263607"
+          - "102344457"
+          - "106879044_174887172"
+          - "104862669_172226853"
+          - "106366353_174226695"
+          - "102344022"
+          - "102344403"
+          - "102815859"
+          - "102816009"
+        excluded_substrings:
+          - "NONE"
+          - "102815859_169535055"
+          - "orig"
+      - name: "val"
+        included_substrings:
+          - "102816756"
+          - "102817140"
+          - "103997586_171030669"
+          - "103997895_171031182"
+          - "104348328_171513363"
+          - "104348361_171513414"
+          #- "105515211_173104185" #no rec filter file annotated
```

Comment:
Thanks, Alex! Will this be permanently excluded?

Reply:
Just excluded for now until we can fix blocking issues and generate a rec filter file. Currently I think scene is missing the entire kitchen counter, lol.